Project.toml

```diff
@@ -7,6 +7,7 @@ version = "0.8.1"
 HTTP = "cd3eb016-35fb-5094-929b-558a96fad6f3"
 JSON = "682c06a0-de6a-54ab-a142-c8b1cf79cde6"
 LibGit2 = "76f85450-5226-5b5a-8eaa-529ad045b433"
+Logging = "56ddb016-857b-54e1-b83d-db4d58db5568"
```

Comment:
The dependency on Logging (and by extension the `using Logging` occurrences) isn't actually necessary, as the basic logging macros—`@info`, `@warn`, and `@error`—are available in Base.

Reply:
Got it, removed the entry and the using statements.